types/react-paginate/index.d.ts

```diff
@@ -1,11 +1,8 @@
-// Type definitions for react-paginate 4.3
+// Type definitions for react-paginate 5.2
 // Project: https://github.com/AdeleD/react-paginate
-// Definitions by: Simon Hartcher <https://github.com/deevus>
-//                 Wouter Hardeman <https://github.com/wouterhardeman>
-//                 pegel03 <https://github.com/pegel03>
-//                 Simon Archer <https://github.com/archy-bold>
+// Definitions by: Yasunori Ohoka <https://github.com/yasupeke>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.8
+// TypeScript Version: 2.9
```

Comment:
Is there a particular reason you want to require consumers of this package to be on 2.9? There's nothing obvious to me in this PR that requires consumers to use TS 2.9, but I might be missing something.